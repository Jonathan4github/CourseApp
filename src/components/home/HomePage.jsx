import React from 'react';
import { Link } from 'react-router-dom';


const HomePage = () => (
  <div className="jumbotron">
    <h1 className="display-4">PluralSight Administration</h1>
    <p>Manage and view available courses on pluralsight</p>
    <p className="lead">
      <Link to="/about" className="btn btn-lg">Learn More</Link>
    </p>
  </div>
);

export default HomePage;
