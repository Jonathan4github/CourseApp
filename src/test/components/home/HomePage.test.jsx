/* eslint-disable */
import HomePage from '../../../components/home/HomePage';


describe('<HomePage /> component test', () => {
  it('renders page header for profile component without crashing', () => {
    const shallowWrapper = shallow(<HomePage />);
    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
