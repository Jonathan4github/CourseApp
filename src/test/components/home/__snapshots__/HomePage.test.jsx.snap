// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> component test renders page header for profile component without crashing 1`] = `
<div
  className="jumbotron"
>
  <h1
    className="display-4"
  >
    PluralSight Administration
  </h1>
  <p>
    Manage and view available courses on pluralsight
  </p>
  <p
    className="lead"
  >
    <Link
      className="btn btn-lg"
      replace={false}
      to="/about"
    >
      Learn More
    </Link>
  </p>
</div>
`;
